css/home-not-logged-in.styl

```diff
@@ -19,7 +19,7 @@ $flex-section
   justify-content: center
   text-align: center
 
-.on-home-page-not-logged-in
+.on-home-page-not-logged-in, .on-home-page-logged-in
```

Comment:
It's confusing to add this to `css/home-not-logged-in.styl`. I wouldn't expect to look for a style for a logged in state in a file named for not logged in. If it's styles that apply to both, then perhaps we should leverage the existing `home-page.styl` file? There's lots there that I imagine isn't being used anymore since both pages have now been rewritten.